src/Core/src/Handlers/Editor/EditorHandler.iOS.cs

```diff
@@ -14,21 +14,24 @@ protected override MauiTextView CreatePlatformView()
 			var platformEditor = new MauiTextView();
 
 #if !MACCATALYST
-			var accessoryView = new MauiDoneAccessoryView(OnDoneClicked, this);
+			var accessoryView = new MauiDoneAccessoryView();
+			accessoryView.SetDoneClicked(OnDoneClicked);
 			platformEditor.InputAccessoryView = accessoryView;
 #endif
 
 			return platformEditor;
 		}
 
+#if !MACCATALYST
```

Comment:
Is this too much negative? 😜 Would `#if iOS` be more readable? Just saying that someone could miss the "!" or, is this planned if we support tvOS or watchOS? 
_Not a mandatory change_

Reply:
Our TFMS are setup a bit different than the rest of the world :-) For the MAUI repository because 98 percent of the IOS code is exactly the same as MACCATALYST we just compile all iOS code for MACCATALYST 
https://github.com/dotnet/maui/blob/main/.nuspec/Microsoft.Maui.Controls.MultiTargeting.targets#L99-L101

 Which works in most cases but then occasionally you end up with stuff like this.